vocales.css

```
@@ -0,0 +1,69 @@
+*{
+    font-family: sans-serif;
+    box-sizing: border-box;
+    font-family: 'Baloo Tamma 2', cursive;    
+}
+body{
+    margin: 0;
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: center;
+    justify-content: center;
+
+}
+
+.nuevo{
+    border: 2px dotted black;
+    margin: 5px;
+    padding: 10px;
+    display: flex;
+    flex-flow: column wrap;
+    align-items: center;
+    justify-content: center;
+}
+.dragndrop p{
+    font-size: 2.1em;
+    text-align: center;
+}
```

esto es del dragndrop
```suggestion
.dragndrop p{
    font-size: 2.1em;
    text-align: center;
    font-weight: 700;
}
```